src/Components/People.js

```diff
@@ -28,10 +32,22 @@ class People extends Component {
       this.setState({ personSearchArray: [] });
     }
   };
-
+  
+  
+  componentDidMount() {
+    this.personSearch();  
+  }
+  
+  
+  personMatch = () => {
+    const searchedPerson = this.state.personSearchArray.filter((person) => {
```

Comment:
We talked about this previously, but great work! We can also use .find() here which will return the same thing, but as an object instead of an array with an object in it